components/inputs/test/input-date-time.visual-diff.js

```diff
@@ -46,8 +46,10 @@ describe('d2l-input-date-time', () => {
 
 	it('basic-focus', async function() {
 		await page.$eval('#basic', (elem) => elem.focus());
-		const rect = await visualDiff.getRect(page, '#basic');
-		await visualDiff.screenshotAndCompare(page, this.test.fullTitle(), { clip: rect });
+		setTimeout(async() => {
+			const rect = await visualDiff.getRect(page, '#basic');
+			await visualDiff.screenshotAndCompare(page, this.test.fullTitle(), { clip: rect });
+		}, 160);
```

Comment:
Not sure if using the sinon fake timers here to quickly advance the time could make these run faster, or if it'll just make them not work at all. 🤷

Reply:
That's a good idea - I'll give it a go